packages/manager/modules/user-account/src/contacts/user-contacts.service.js

```diff
@@ -0,0 +1,306 @@
+import set from 'lodash/set';
+import without from 'lodash/without';
+
+angular
+  .module('UserAccount')
+  .service('UserAccount.services.Contacts', function UserAccountContactsService(
+    $rootScope,
```

Comment:
I wonder how in a module, we can bundle this service without using any import or exports

Reply:
Sorry this must have got in when i refactored. Have removed it